templates/flavors/azure-cni-v1/patches/azure-cluster-template.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureCluster
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  resourceGroup: "${AZURE_RESOURCE_GROUP:=${CLUSTER_NAME}}"
+  location: "${AZURE_LOCATION}"
+  subscriptionID: ${AZURE_SUBSCRIPTION_ID}
+  networkSpec:
+    vnet:
+      name: "${AZURE_VNET_NAME:=${CLUSTER_NAME}-vnet}"
+    subnets: # single NIC per nvet
```

```suggestion
    subnets: # single NIC per vnet
```

although I'm not sure I get the significance of this comment, we're not configuring NICs here but subnets?

---

Sorry for the confusion. The comment was a meta information for me. I have removed it.